sigar_darwin.go

```diff
@@ -422,6 +422,10 @@ func task_info(pid int, info *C.struct_proc_taskallinfo) error {
 	return nil
 }
 
+func (self *SystemDistribution) Get() error {
+	return notImplemented()
```

Comment:
Yeah, I was surprised to see that `notImplemented` panics when it could just return an error. That seems like something that should be nuked from the codebase.

I'll do that.